students/DanCornutt/session05/dict_set_lab2.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+"""dict and set lab with comprehension
+
+
+"""
+def c_dict():
+    """dict of chris's favorites.
+
+    """
+    food_prefs = {"name": "Chris",
+                  "city": "Seattle",
+                  "cake": "chocolate",
+                  "fruit": "mango",
+                  "salad": "greek",
+                  "pasta": "lasagna"}
+    return food_prefs
+
+def chris_text(food_prefs):
+    """returns text about chris.
+
+    """
+    body = """Chris is from {}, and he likes {} cake, {} fruit, {} salad,
+    and {} pasta""".format(
+        food_prefs['city'], food_prefs['cake'],
+        food_prefs['fruit'], food_prefs['salad'],
+        food_prefs['pasta'])
+    return body
+
+
+def hex_dict():
+    """creates dictionary of integers and equivalent hexidecimal numbers
+
+    """
+    h_dict = dict(list(enumerate([hex(i) for i in range(15)])))
```

Comment:
no need for the list() here -- enumerate already is an iterable.

And can you do it with a dict comp?